torchao/quantization/pt2e/qat_utils.py

```diff
@@ -888,7 +888,7 @@ def _fold_conv_bn_qat(m: GraphModule) -> GraphModule:
             and node.args[0].op == "get_attr"
             and node.args[1] == 1
             and torch.nn.modules.batchnorm.BatchNorm2d
```

Comment:
I think this needs to be a string instead according to this:

<img width="1095" height="295" alt="Image" src="https://github.com/user-attachments/assets/74f0a097-bec6-4a5f-a860-6213879ab159" />